prov/verbs/src/ep_rdm/verbs_rdm.h

```diff
@@ -196,7 +196,22 @@ fi_ibv_rdm_tagged_zero_request(struct fi_ibv_rdm_tagged_request *request)
 void fi_ibv_rdm_tagged_print_request(char *buf,
 				     struct fi_ibv_rdm_tagged_request *request);
 
+typedef short fi_ibv_rdm_buf_status_t;
+#define BUF_STATUS_FREE		((fi_ibv_rdm_buf_status_t) 0)
+#define BUF_STATUS_BUSY		((fi_ibv_rdm_buf_status_t) 1)
+#define BUF_STATUS_RECVED	((fi_ibv_rdm_buf_status_t) 2)
```

Comment:
enum has 32-bit size by default, I replaced it by short and corresponding definitions from performance reason. Before this series, buffer status has generally local nature (excepting releasing). Now, it's a part of protocol which is sending every time with header and users data. So, I would say, service data of a buffer (including status) is a things of lower level protocol then eager protocol.

I'll replace typedef by uint16_t.